### PR TITLE
Fix: Placeholder values of column width shouldn't cascade to mobile [ED-4664]

### PR DESCRIPTION
**Pull request by YitzhakAvm**

### CODE REVIEW

File: assets/dev/js/editor/controls/base-data.js [hunk 1]
```diff
@@ -216,9 +216,12 @@ ControlBaseDataView = ControlBaseView.extend( {
 		const parent = this.getResponsiveParentView();
 
 		if ( parent ) {
-			const placeholder = parent.preparePlaceholderForChildren();
+			const name = this.model.get( 'name' ),
+				placeholder = '_inline_size_mobile' === name ?
```
Comment (matipojo) [inline assets/dev/js/editor/controls/base-data.js:220]: It shouldn't be hardcoded. you can configure the control like

```
$this->add_responsive_control(
			'_inline_size',
			[
				'label' => __( 'Column Width', 'elementor' ) . ' (%)',
				'type' => Controls_Manager::NUMBER,
				'responsive_placeholder' => 100,
```

and here you will use this config

**Review by matipojo — CHANGES_REQUESTED** [review]

File: includes/controls/date-time.php [hunk 2]
```diff
@@ -62,7 +62,7 @@ public function content_template() {
 		<div class="elementor-control-field">
 			<label for="<?php $this->print_control_uid(); ?>" class="elementor-control-title">{{{ data.label }}}</label>
 			<div class="elementor-control-input-wrapper">
-				<input id="<?php $this->print_control_uid(); ?>" placeholder="{{ data.placeholder }}" class="elementor-date-time-picker flatpickr" type="text" data-setting="{{ data.name }}">
+				<input id="<?php $this->print_control_uid(); ?>" placeholder="{{ elementor.getPanelView().currentPageView.children.findByModelCid( data._cid )?.getControlPlaceholder() }}" class="elementor-date-time-picker flatpickr" type="text" data-setting="{{ data.name }}">
```
Comment (matipojo) [inline includes/controls/date-time.php:65]: Find a shorter way or create a method for it.
@kobizz can you advise?

File: includes/controls/dimensions.php [hunk 3]
```diff
@@ -104,18 +104,19 @@ public function content_template() {
 								// PHPCS - the variable $dimension_key is a plain text.
 								echo $dimension_key; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 							?>" placeholder="<#
-								if ( _.isObject( data.placeholder ) ) {
-									if ( ! _.isUndefined( data.placeholder.<?php
+								placeholder = elementor.getPanelView().currentPageView.children.findByModelCid( data._cid )?.getControlPlaceholder();
```
Comment (matipojo) [inline includes/controls/dimensions.php:107]: Like above in all places

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]
Temporary Approved, we will fix it later.

File: includes/controls/date-time.php (hunk 2 above)
Comment (kobizz) [reply includes/controls/date-time.php:65]: You can pass the model as a template helper in `assets/dev/js/editor/controls/base.js` under `templateHelpers()` method, then you can simply call `{{ model.getControlPlaceholder() }}`

**Pull request merged**